lib/pages/chat/events/message.dart

```diff
@@ -206,7 +207,9 @@ class Message extends StatelessWidget {
                                       crossAxisAlignment:
                                           CrossAxisAlignment.start,
                                       children: [
-                                        ownMessage || event.room.isDirectChat
+                                        ownMessage ||
+                                                event.room.isDirectChat ||
+                                                !showDisplayName
```

Comment:
Deserve a method

Reply:
.

Reply:
there is already one I forgot to use it mb